src/etc/schema.graphqls

```diff
@@ -28,4 +29,22 @@ type StoreConfig {
     reviews_are_enabled: Boolean @doc(description: "Are reviews enabled")
     guest_checkout: Boolean  @doc(description: "Is guest checkout enabled")
     is_email_confirmation_required: Boolean  @doc(description: "Is customer email confirmation enabled")
+    address_lines_quantity: Int @doc(description: "Number of address lines in forms")
```

Comment:
All the subsequent fields after `address_lines_quantity` are unnecessary, they should already be present on the `StoreConfig` type.